index.html

```diff
@@ -146,11 +146,17 @@
     }
   },
   registerData: function(d) {
-    var assignee = (d.assignee) ? d.assignee.login : inlineApp.NOT_ASSIGNED;
-    if (!inlineApp.data[assignee]) {
-      inlineApp.data[assignee] = [];
+    if (d.assignees.length == 0) {
+      inlineApp.data[inlineApp.NOT_ASSIGNED].push(d)
+    } else {
+      for (var i = 0; i < d.assignees.length; i++) {
+        var assignee = d.assignees[i].login
+        if (!inlineApp.data[assignee]) {
+          inlineApp.data[assignee] = [];
+        }
+        inlineApp.data[assignee].push(d);
```

Comment:
`assigned xxx / yyy` on the top of table, should counts unique issues. But issues which is assigned for multiple assignees seem to be double counted.
see around here.
https://github.com/opt-tech/issues-by-assignee/blob/master/index.html#L193